modules/core/include/opencv2/core/openvx/ovx_defs.hpp

```diff
@@ -20,6 +20,12 @@
 #define IVX_USE_OPENCV
 #include "ivx.hpp"
 
+namespace cv
+{
+// Get common thread local OpenVX context
+CV_EXPORTS_W ivx::Context& getThreadOpenVXContext();
```

Comment:
Probably `getOpenVXContext()` name is better.
Also it is better to place this into `cv::ovx` namespace.